server/src/main/java/io/crate/expression/operator/any/AnyNeqOperator.java

```diff
@@ -73,7 +73,7 @@ protected Query refMatchesAnyArrayLiteral(Function any, Reference probe, Literal
             }
             andBuilder.add(fromPrimitive, BooleanClause.Occur.MUST);
         }
-        Query exists = IsNullPredicate.refExistsQuery(probe, context, false);
+        Query exists = IsNullPredicate.refExistsQuery(probe, context);
```

Comment:
[So far the `probe` argument was never an array type](https://github.com/crate/crate/issues/16299) so the changes in `IsNullPredicate.refExistsQuery` does not affect this execution path.

Reply:
Also the calls from `ArrayUpperFunction` is about to be removed from other PR.